de.capypara.FieldMonitor.json

```
@@ -0,0 +1,319 @@
+{
+    "id": "de.capypara.FieldMonitor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "48",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "24.08",
+            "add-ld-path": ".",
+            "no-autodownload": false,
+            "autodelete": false
+        },
+        "org.freedesktop.Platform.openh264": {
+            "directory": "lib/openh264",
+            "version": "2.4.1",
+            "add-ld-path": "."
+        }
```

```suggestion
        }
```

this is installed by the runtime, you don't need this